index.html

```diff
@@ -67,13 +68,38 @@
 
 let population = parseInt(document.getElementById("population").value);
 let previous_population = parseInt(document.getElementById("population").value);
+let previous_initial_rate = parseInt(document.getElementById("initialInfectionRate").value);
```

Comment:
do you need these new variables or can you not just use the current population and initial infection rate?

Reply:
generally try to use what's already there.

Reply:
This was used when I first try to update the values as soon as the user has typed in a number. This is indeed no longer needed, I will change that, thanks.